src/System.IO.Abstractions.TestingHelpers/MockFileData.cs

```diff
@@ -27,6 +27,14 @@ public class MockFileData
             Attributes = (FileAttributes)(-1),
         };
 
+        internal static readonly MockFileData NewObject = new MockFileData(string.Empty)
+        {
+            CreationTime = DateTime.Now,
+            LastWriteTime = DateTime.Now,
+            LastAccessTime = DateTime.Now,
+            Attributes = FileAttributes.Normal,
```

Comment:
That already works now, doesn't it?

Reply:
Yes you are right! It looks like I had a false assumption based on a test I made. If you do it this way `Attributes` is indead -1
```
var fi = fileSystem.FileInfo.FromFileName(file); //File does not exist
using var stream = fi.Create(); //File exists
var attr = fileSystem.File.GetAttributes(file); //FileAttributes.Normal
var attr2 = fi.Attributes; // -1
```
which is correct I assume. With a `FileInfo.Refresh()` before accessing the `Attributes` it works as expected.